src/lib/holocene/data-encoder-settings.svelte

```diff
@@ -84,14 +84,20 @@
         <h3 data-testid="data-encoder-title" class="text-xl">Codec Server</h3>
       </div>
       <p class="text-sm">
-        The Codec Server uses a remote codec endpoint to decrypt your payloads,
-        ensuring that Temporal never sees your data. A remote codec endpoint can
-        be set at the {namespaceOrCluster}-level, or locally in your browser.
+        A <a
+          rel="noreferrer"
+          target="_blank"
+          href="https://docs.temporal.io/dataconversion#codec-server"
+          class="text-blue-700 underline decoration-blue-700">Codec Server</a
+        >
+        decodes your payloads, ensuring that Temporal never sees your data. A Codec
```

Comment:
```suggestion
        decodes your data. A Codec
```

@Alex-Tideman @aarohib because decoding may not decrypt?